lib/classes/PluginManager.js

```diff
@@ -215,6 +220,38 @@ class PluginManager {
     );
   }
 
+  getLocalPluginsPaths() {
+    const fs = require('fs');
+
+    const pluginsPaths = this.localPluginsPaths.map(({ entryFileRealPath, originalPath }) => {
+      // Scenario 1: a file was specified WITH extension, eg "./folder/my-plugin.js"
+      if (originalPath.endsWith('.js')) {
+        return entryFileRealPath;
+      }
+
+      let realPath;
+      try {
+        realPath = fs.realpathSync(originalPath);
+      } catch (err) {
+        realPath = fs.realpathSync(`${originalPath}.js`);
+      }
```

Comment:
I'm still not convinced we need another fs check here. Wouldn't it be better to go as follows:

```javascript
const absoluteInputPath = path.resolve(serviceDir, inputPath);
if ((absoluteInputPath + path.extname(realPath)) === realPath) // assume realPath (file)
if (realPath.startsWith(absoluteInputPath + path.sep)) // assume absoluteInputPath (dir)
```